src/components/main-page/Location/Location.tsx

```diff
@@ -27,23 +27,23 @@ export function Location() {
               <Icon className={styles.icon} icon={mapMarker} />
             </div>
             <div className={styles.headingWrapper}>
-              <p className={styles.bold}>Lokalizacja 1</p>
-              <p className={styles.bold}>Lokalizacja 2</p>
-              <p>Lokalizacja 3</p>
+              <p className={styles.bold}>Centrum Wykładowe</p>
```

Comment:
te trzy linijki powinny mieć niełamliwą spację - lepiej wtedy wyglada na mobilce
```suggestion
              <p className={styles.bold}>Centrum&nbsp;Wykładowe</p>
              <p className={styles.bold}>Politechniki&nbsp;Poznańskiej</p>
              <p>Kampus&nbsp;Piotrowo, Poznań</p>
```